extensions/python/syntaxes/MagicRegExp.tmLanguage.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "MagicRegExp",
+	"name": "Python",
```

Comment:
This one should say something else -- that's a syntax for Python regexps. The name isn't that important, it can be "Regular Expressions (Python)"